packages/components/src/templates/teaser-with-icon.stories.ts

```diff
@@ -0,0 +1,164 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+/**
+ * ```
+ * ```
+ */
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Teaser Two',
```

Comment:
I'm really not happy with this change. Currently it's desriptive and as you now realized, there are sometimes templates, that don't have a counterpart on component side (e. g. Quote). Calling it "Teaer One" and "Teaser Two" feels completely random. Why not calling it what we see here?

So I would absolutely vote against this change.

Reply:
Please ignore this PR this is not going to happen. I agree with you that was not intended to be commited